color/p5js/colm/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Filip Czekajlo
+author-url: https://github.com/filipczekajlo
+title: colm
+tags: [P5.js, color, Law of Good Gestalt]
```

Comment:
Please add the tags as a list in the frontmatter  

https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-P5js-Sketches#markdown-and-front-matter  

tags: 
  - P5.js
  - color
  - Law of Good Gestalt